conformance/apis/v1alpha1/doc.go

```diff
@@ -23,17 +20,10 @@ limitations under the License.
 // are written in Golang and execute the conformance test suite as a Golang
 // library.
 //
-// Note that currently all sub-packages are considered "experimental" in that
-// they aren't intended for general use or to be distributed as part of a
-// release so there is no way to use them by default when using the Golang
-// library at this time. If you don't know for sure that you want to use these
-// features, then you should not use them. If you would like to opt into these
-// unreleased features use Go build tags to enable them, e.g.:
-//
-//   $ GOFLAGS='-tags=experimental' go test ./conformance/... -args ${CONFORMANCE_ARGS}
+//   $ go test ./conformance/... -args ${CONFORMANCE_ARGS}
 //
-// Please note that everything here is considered experimental and subject to
-// change. Expect breaking changes and/or complete removals if you start using
-// them.
+// While the experiment build tag is removed, please note that everything here
```

Comment:
I wonder if we really need to mention the experimental build tag here. From a user perspective, this preamble can be meaningless, as such a tag does not exist anymore.

Reply:
+1, vote for keeping as is

Reply:
I'd probably also not mention the experimental build tag here.

Reply:
good point, let me revert this change.